hack/make-rules/test-cmd-util.sh

```diff
@@ -1542,16 +1541,16 @@ run_non_native_resource_tests() {
   kube::log::status "Testing kubectl non-native resources"
   kube::util::non_native_resources
 
-  # Test that we can list this new third party resource (foos)
+  # Test that we can list this new CustomResource (foos)
```

Comment:
Why not custom resource definition? (not compulsory, just curious what's the formal name we are using)

Reply:
+1. It's better to use the full name.

Reply:
This does a `kubectl get foos` which lists the _instances_ of our custom type i.e. it lists the "custom resources". [This](https://github.com/nikhita/kubernetes/blob/694630724a1e5036d35ed9c12dc97dd2989d15d6/hack/testdata/CRD/foo.yaml) is an instance of type `Foo` .

For the `CustomResourceDefinition` (where we register the type `Foo`), we need to do `kubectl get crd`, which is done [here](https://github.com/nikhita/kubernetes/blob/694630724a1e5036d35ed9c12dc97dd2989d15d6/hack/make-rules/test-cmd-util.sh#L1480).

Reply:
So it seems to be mixed usage or inconsistency?

Reply:
CRDs and CRs are two different things.